pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -84,6 +85,10 @@ const (
 	gceHcUnhealthyThreshold = int64(5)
 )
 
+var (
+	diskNotFoundErr = errors.New("disk is not found")
```

Comment:
No the best style to have a global. Make a type out of it and use a type select for a check.